arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -236,11 +248,43 @@
           "nextLinkName": null
         }
       }
+    },
+    "/providers/Microsoft.Sql/operations": {
```

Comment:
is this API documented? works? it looks like each of the swagger files include the same api "/providers/Microsoft.Sql/operations", shouldn't it be different for firewallRules.json, replicationLinks.json, and sql.core.json?